### PR TITLE
docs: DEV-2941: Release Notes for LSE 2.3

### PR DESCRIPTION
**Pull request by anithamuthu2022**

### CODE REVIEW

**Comment by anithamuthu2022** [comment]
@makseq @niklub feel free to pull this branch down and make changes accordingly.

File: docs/source/guide/release_notes.md [hunk 1]
```diff
@@ -12,10 +12,94 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+    - Manual regions grouping (drag-and-drop)
+    - Hiding/locking regions
+    - Prediction indication on the region
+    - Region editor (Image segmentation)
+    - Relations list
+    - Panels drag-and-drop
+    - Panels snap-to-side
+    - Panels resizing
+
+- [internal] Show jobs by projects and organizations on **/django-rq** page [DEV-2898].
```
Comment (makseq) [inline docs/source/guide/release_notes.md:37]: We don't need to publish [internal] things

File: docs/source/guide/release_notes.md [hunk 2]
```diff
@@ -12,10 +12,94 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+    - Region editor (Image segmentation)
+    - Relations list
+    - Panels drag-and-drop
+    - Panels snap-to-side
+    - Panels resizing
+
+- [internal] Show jobs by projects and organizations on **/django-rq** page [DEV-2898]. 
+- Add the ability to sustain the collapse state of the label across tasks [DEV-2755].
+- This release provides consistent ordering of the label groups [DEV-2755].
+- Implement new columns for comment management in the **Data Manager**: **Comment count**, **unresolved comment count**, **comment authors**, and **last comment date** [DEV-2672].
```
Comment (makseq) [inline docs/source/guide/release_notes.md:40]: I would group everything regarding comments into one group and start with `Introduce the [**Comments**](comments_annotations.html) section for annotation editor. ...`

**Review by makseq — CHANGES_REQUESTED** [review]

**Comment by anithamuthu2022** [comment]
@makseq I've addressed your feedback for attention. Thank you!

**Comment by anithamuthu2022** [comment]
@makseq could you confirm if **Notifications** related entries should be removed from this PR?
Reference: https://heartex.slack.com/archives/C03DQ8U76PP/p1659395055710989?thread_ts=1659394615.296519&cid=C03DQ8U76PP

**Comment by anithamuthu2022** [comment]
@makseq I've removed `Comments` and `Notifications` entries in LSE 2.3 section as per https://heartex.slack.com/archives/C03DQ8U76PP/p1659395055710989?thread_ts=1659394615.296519&cid=C03DQ8U76PP.

**Comment by anithamuthu2022** [comment]
@makseq could you review and approve this PR for today's release.

**Review by makseq — APPROVED** [review]

**Comment by makseq** [comment]
> @makseq could you confirm if **Notifications** related entries should be removed from this PR? Reference: https://heartex.slack.com/archives/C03DQ8U76PP/p1659395055710989?thread_ts=1659394615.296519&cid=C03DQ8U76PP

Yes, notifications are not implemented and postponed to the next releases

**Comment by anithamuthu2022** [comment]
@niklub @joealfaro could you please review and approve this PR? I got the information from Max to take your approvals before the PR merge.

**Review by joealfaro — APPROVED** [review]

File: docs/source/guide/release_notes.md [hunk 3]
```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+- Pin and create shared view of relevant projects in your workspace: Allow users to [pin or unpin projects](manage_users.html#More-menu-to-Pin-or-Unpin-and-Duplicate-projects) to more easily filter and find projects of interest. This feature allows you to pin and create a shared view of relevant projects in your workspace. Administrators now have the ability to pin and unpin projects for quicker reference by team members, whether they are projects the team is currently working on, or projects of interest. All team members will have a view of pinned projects to collaborate on in their workspace [DEV-2069].
+- Zoom, align, and pixelate images for annotation: Annotators can zoom and expand images they are labeling within the viewport of the task, set the image to be positioned vertically (top, center, bottom) and horizontally (left, center, right) to find the most convenient position. Once an image is zoomed in, they can also choose to pixelate the image if it is blurry for more precise annotations [DEV-2504].
+- Add the ability to sustain the collapse state of the label across tasks [DEV-2755].
```
Comment (niklub) [inline docs/source/guide/release_notes.md:27]: Combine 27,28 into one line

File: docs/source/guide/release_notes.md [hunk 4]
```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+- This release provides consistent ordering of the label groups [DEV-2755].
+- Enforce the change in the rotate anchor of bounding boxes around image boundaries [DEV-2671].
+- Allow annotators and reviewers to filter view of transcriptions by **author name** [DEV-2669].
+- This release improves the polygon point removal when you create a new polygon. In addition to the double-click and alt-click, now you can use the undo hotkeys (`ctrl/cmd+z`) to remove the point you set or redo it by clicking the `ctrl/cmd+shift+z` hotkeys [DEV-2576].
+- Migrate Label Studio Enterprise and Label Studio Frontend to Playwright for end-to-end activity [DEV-2481].
```
Comment (niklub) [inline docs/source/guide/release_notes.md:32]: remove this

File: docs/source/guide/release_notes.md [hunk 5]
```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+- Pin and create shared view of relevant projects in your workspace: Allow users to [pin or unpin projects](manage_users.html#More-menu-to-Pin-or-Unpin-and-Duplicate-projects) to more easily filter and find projects of interest. This feature allows you to pin and create a shared view of relevant projects in your workspace. Administrators now have the ability to pin and unpin projects for quicker reference by team members, whether they are projects the team is currently working on, or projects of interest. All team members will have a view of pinned projects to collaborate on in their workspace [DEV-2069].
+- Zoom, align, and pixelate images for annotation: Annotators can zoom and expand images they are labeling within the viewport of the task, set the image to be positioned vertically (top, center, bottom) and horizontally (left, center, right) to find the most convenient position. Once an image is zoomed in, they can also choose to pixelate the image if it is blurry for more precise annotations [DEV-2504].
```
Comment (niklub) [inline docs/source/guide/release_notes.md:26]: is there a link to rich description here?

File: docs/source/guide/release_notes.md [hunk 6]
```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+- This release improves the polygon point removal when you create a new polygon. In addition to the double-click and alt-click, now you can use the undo hotkeys (`ctrl/cmd+z`) to remove the point you set or redo it by clicking the `ctrl/cmd+shift+z` hotkeys [DEV-2576].
+- Migrate Label Studio Enterprise and Label Studio Frontend to Playwright for end-to-end activity [DEV-2481].
+
+### Bug fixes
+
+Label Studio 2.3 includes the following bug fixes:
+
+- Fixed an issue when the text captured by a region was not displayed in the **Details** panel [DEV-2958].
+- Resized the left panels in **Quick View** mode to work consistently when expanded to the right in **Label** or **Review** stream, and stop after the maximum width was reached [DEV-2952], [DEV-2943].
+- Fixed an issue when grouping by empty label (**Group by Label**) caused the Label Studio App to crash [DEV-2942].
```
Comment (niklub) [inline docs/source/guide/release_notes.md:40]: I see this ticket and other ones refer to the Outliner feature that is not a part of this release. Please remove everything related to the Outliner Epic

File: docs/source/guide/release_notes.md [hunk 7]
```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
...
+- [DEV-2639].
+- Fixed the behavior of the drop-down menu that failed to group when the organization was not activated [DEV-2639].
+- Updated annotation failed to display in the **Review Annotations** stream [DEV-2533].
+- Fixed an issue where **Quick View** failed to open when the user attempted to copy-paste its URL to another tab [DEV-2526].
+- Unfinished polygon region failed to auto-completed when the user moved it [DEV-2514].
+- Multiple labels failed to show within **Regions** [DEV-2503].
+- Unfinished polygons were saved as a draft and remained in an open state if left unclosed [DEV-2432].
+- Fixed broken default page number for non-admin accounts on **Projects** page [DEV-2335].
+- **Annotation History** item always displayed the current state of the annotation [DEV-2301].
+- Fixed a vulnerability issue against blind server-side request forgery (SSRF) on data import [DEV-2234].
```
Comment (niklub) [inline docs/source/guide/release_notes.md:64]: Group all security fixes into a separate section

**Review by niklub — CHANGES_REQUESTED** [review]

File: docs/source/guide/release_notes.md (hunk 5 above)
Comment (anithamuthu2022) [reply docs/source/guide/release_notes.md:26]: No.

File: docs/source/guide/release_notes.md (hunk 6 above)
Comment (anithamuthu2022) [reply docs/source/guide/release_notes.md:40]: I deleted entries based on https://heartex.atlassian.net/browse/DEV-1544. @niklub please let me know if there are any further changes in this regard.

**Pull request closed without merging**